client/rpc-core/src/types/call_request.rs

```diff
@@ -49,4 +50,26 @@ pub struct CallRequest {
 	/// EIP-2718 type
 	#[serde(rename = "type")]
 	pub transaction_type: Option<U256>,
+	/// Call state override
```

Comment:
[geth](https://geth.ethereum.org/docs/interacting-with-geth/rpc/ns-eth) implements the `state_override` as a third parameter to `eth_call`, should we perhaps follow that and create a new version `5` of `call` @tgmichel ?

Reply:
The goal is to support tools relying on geth behavior, so yes we should mimic the geth rpc api.

But there is no versioning on RPC methods, I think you are confusing with the versioning of runtime apis (that we don't need to touch here).

Reply:
So, you can directly modify the `fn call` signature, and that's it. As long as the parameter is an `Option`, jsonrpsee will automatically set it to `None` if it's not provided.